core/server/master/src/main/java/alluxio/master/metrics/CloudCostUtils.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master.metrics;
+
+import alluxio.master.file.DefaultFileSystemMaster.Metrics.UFSOps;
+
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Map;
+
+/**
+ * This class contains utilities to compute cloud cost savings.
+ */
+public final class CloudCostUtils {
+  private CloudCostUtils() {}  // prevent instantiation
+
+  public static final Map<UFSOps, Double> S3COSTMAP =
```

Comment:
What are all the units for these values?

Also, should each of these maps be pointing to a website to where the costs are retrieved? How often do they need to be updated?

Reply:
done